gpt.go

```diff
@@ -473,7 +473,7 @@ func stringToGuid(guid string) (res [16]byte, err error) {
 			case 'F', 'f':
 				bt |= 15 << shift
 			default:
-				err = fmt.Errorf("BAD guid char: ", i+pos, ch)
```

Comment:
I propose format message:
"BAD guid char at pos %d: '%c'" - surround char in single quotas